### ISSUE TITLE
Hiding

### ISSUE THREAD

**Issue by agahEbrahimi** [chjj/blessed#345]
Hi, 

Could you hide the terminal (as in not show up on the screen or the toolbar) using blessed? If yes, how?

Thank you very much

**Comment by LoganDark** [comment]
No. This is something the terminal would have to support natively.
